modules/features/nova/src/main/kotlin/au/com/shiftyjelly/pocketcasts/nova/CatalogFactory.kt

```diff
@@ -72,6 +73,28 @@ internal class CatalogFactory(
         },
     )
 
+    fun inProgressEpisodes(data: List<NovaLauncherInProgressEpisode>) = Catalog(
+        id = "ContinueListening",
+        label = context.getString(LR.string.nova_launcher_continue_listening),
+        type = CatalogType.CONTINUE,
+        items = data.map { episode ->
+            CatalogItem.Base(
+                id = episode.id,
+                intent = context.launcherIntent,
```

Comment:
This intent is to be clarified. Right now I used launching the app as a default.